valhalla/sif/edgelabel.h

```diff
@@ -70,24 +70,24 @@ class EdgeLabel {
             const TravelMode mode,
             const uint32_t path_distance,
             const uint8_t restriction_idx,
+            const bool not_thru_pruning,
```

Comment:
I added both `not_thru_pruning` and `destonly_pruning` as first-class citizens to EdgeLabel, also timedistancematrix wants to look at them I think